redis/src/types.rs

```diff
@@ -749,6 +749,12 @@ impl From<(ErrorKind, &'static str, String)> for RedisError {
     }
 }
 
+impl From<(String, Option<String>)> for RedisError {
```

Comment:
this is just for testing right? I prefer to keep the `make_extension_error` function, to make this explicit.

Reply:
Yes, this is done only for testing purposes.

The `make_extension_error` function has to be exposed in order for it to be usable in the tests.

Additionally, all other variants of the `ErrorRepr` enum are already covered by `impl From<...> for RedisError` functions, so this addition simply completes the set.

Reply:
Right, so I prefer that the `make_extension_error` function be exposed than to add this, because I want it to be explicit that this creates an extension error.

Reply:
or even simpler, just add these tests to `types.rs` instead of the tests suite.